src/Microsoft.ML.Data/Prediction/IPredictionTransformer.cs

```diff
@@ -2,22 +2,23 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.Internal.Calibration;
-using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace Microsoft.ML.Runtime
 {
     public interface IPredictionTransformer<out TModel> : ITransformer
         where TModel : IPredictor
+    {
+        TModel Model { get; }
+    }
+
+    public interface IClassicPredictionTransformer<out TModel> : IPredictionTransformer<TModel>
```

Comment:
>IClassicPredictionTransformer [](start = 21, length = 29)

I sort of feel like an interface called `IClassicPredictionTransformer` needs some XML comment on it. All public interfaces should, but especially one with the word "classic" in the name.

I am choosing to interpret this interface as it gives me classic coke whenever I use it.